docs/runbooks/VirtOperatorRESTErrorsBurst.md

```diff
@@ -2,24 +2,53 @@
 
 ## Meaning
 
-More than 80% of the rest calls in virt-operator pods failed in the last 5 minutes. 
+More than 80% of REST calls in `virt-operator` pods failed in the last 5 minutes. 
```

Comment:
For consistency: "This alert fires when more than 80% ....  This alert usually indicates that the `virt-operator` pods cannot connect to the API server."